### ISSUE TITLE
Issue with Image Zoom

### ISSUE THREAD

**Issue by nfon-christophe** [saga-nfon/adapt-nfon-theme#5]
As discussed in our call earlier this week, the Image Zoom is not working properly. Images which are not square are distorted (see screenshot). Could we do the image zoom as a lightbox popup?

![Screenshot 2021-01-29 121535](https://user-images.githubusercontent.com/54438034/106274164-bc778f80-622b-11eb-8dbf-964ad2833bcf.png)
![Screenshot 2021-01-29 121505](https://user-images.githubusercontent.com/54438034/106274166-bd102600-622b-11eb-8307-aca952aa2fe3.png)

**Comment by hardik-sagacity** [comment]
The "graphic" is a core component of the adapt framework. The only way to modify existing components from the theme is through templates which allow modifying the temple HTML code but adding a javascript was a challenge.

earlier we had a CSS rule applied to the image when the user clicks and holds the cursor onto the image. which we achieved using only CSS :active selector.

now I have extended the core graphic component by overriding the graphic template by adding a new file(graphic.hbs) in the theme template. which allows me to change the HTML markup of the core graphic component.

I have added an onclick method on the image container which toggles the full-screen class on click.

now the image is fit to the container and does not distort.

The user can click on the image to close the popup.
